tests/baker_recipes/hr.py

```diff
@@ -0,0 +1,25 @@
+from model_bakery.recipe import Recipe, foreign_key
+
+from .units import department
+
+# Recipes for the `hr.Person` objects
+
+person = Recipe("hr.Person", last_name="Paduszyński", first_name="Kamil")
+
+
+# Recipes for the `hr.Status` objects
+
+status = Recipe("hr.Status", name="employee", abbr="E")
+
+
+# Recipes for the `hr.Contract` objects
+
+
+contract = Recipe(
+    "hr.Contract",
+    person=foreign_key(person),
+    status=foreign_key(status),
+    department=foreign_key(department),
+)
+
+__all__ = ["contract", "person", "status"]
```

Comment:
Insert extra line before `__all__`.
```suggestion

__all__ = ["contract", "person", "status"]
```